File: Easy/3370. Smallest Number With All Set Bits/Solution.py
class Solution(object):
    def smallestNumber(self, n):
        """
        :type n: int
        :rtype: int
        """
        result = 0
        bit_position = 0

        while n > result:
            result += 2 ** bit_position
            bit_position += 1

        return result

File: Easy/3370. Smallest Number With All Set Bits/Solution.cs
public class Solution {
    public int SmallestNumber(int n) {
        int result = 0;
        int bitPosition = 0;

        while (n > result) {
            result += (int) Math.Pow(2, bitPosition);
            bitPosition++;
        }
        return result;
        
    }
}